paper/paper.md

```diff
@@ -26,18 +29,60 @@ affiliations:
     index: 3
   - name: Division of Epidemiology and Biostatistics, School of Public Health, University of California, Berkeley
     index: 4
-date: 01 September 2020
+date: 03 February 2021
 bibliography: paper.bib
 ---
 
```

Comment:
High-level, the contents of the paper look good. Some general changes would be
* naming + citing exactly the covariance matrix estimators that are already implemented (stating the version number of the package for clarity)
* discussing more the visualization functionality, with a simple example. for example, we could grab the `README` example, mention the estimators that we select over, and then display one of a table and the nice plots that shows the performance within estimator families.
* in my most recent experience with submitting to JOSS, the editor asked that we re-organize the paper into short sections (as in https://github.com/nhejazi/txshift/blob/master/paper/paper.md). i think this is a good way to go, so that we have short paragraphs describing what `cvCovEst` does, what the problem is, and how we solve the problem (with technical details relegated to the methods paper)

Reply:
Done! See most recent comments.